pkg/agent/containerd/config_linux.go

```diff
@@ -57,6 +57,7 @@ func setupContainerdConfig(ctx context.Context, cfg *config.Node) error {
 		DisableCgroup:         disableCgroup,
 		IsRunningInUserNS:     isRunningInUserNS,
 		PrivateRegistryConfig: privRegistries.Registry(),
+		ExtraRuntimes:         findNvidiaContainerRuntimes(nil),
```

Comment:
passing in a `nil` here just so that the tests can pass in an alternative implementation seems weird, but I don't know what the convention is for doing something like this - @briandowns?

Reply:
Code shouldn't ever know it's under test. You'll then have the observer effect of the implementation.

Reply:
fs.FS support is not strictly for tests, rather for future compatibility purposes. I considered using os.DirFS("/") here instead of nil, does that make more sense?

Reply:
I'd prefer that over `nil`.